public/index.html

```diff
@@ -2,6 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link
```

Comment:
Can you investigate if this can be imported differntly without using a link in the index.html?

Reply:
Maybe using Google's Webfontloader (https://github.com/typekit/webfontloader), with an <script> like this:
<script>
  WebFont.load({
    google: {
      families: ['Roboto Condensed', 'Merriweather']
    }
  });
</script>

Or importing them in a css file

Reply:
Yeah, I think this one it's better!